fetch.bs

```diff
@@ -8615,10 +8681,17 @@ method steps are:
    <li><p><a for=/>Resolve</a> <var>p</var> with <var>responseObject</var>.
   </ol>
 
+ <li><p>Set <var>controller</var> to the result of calling <a for=/>fetch</a> given
+ <var>request</var> with <a for=fetch><i>processResponse</i></a> set to <var>processResponse</var>,
+ <a for=fetch><i>processRequestBodyChunkLength</i></a> set to <var>processRequestBodyChunkLength</var>,
+ and <a for=fetch><i>processRequestEndOfBody</i></a> set to <var>processRequestEndOfBody</var>.
+
  <li><p>Return <var>p</var>.
 </ol>
 </div>
 
+TEMPORARY <dfn id=event-fetchobserver-progress event for=FetchObserver><code>progress</code></dfn>
```

Comment:
Need to define the event somewhere properly. (potentially taking the event definitions from XHR?)